mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
This prevents the free link recognition?

Is this an issue for text emails?  It seems to me that our link formatting should do the correct thing with free links in a text formatter (that is render it as itself, it gets processed, but shouldn't be changed).

If you are doing html emails, that's a different story, although it still isn't clear when they wold want this difference in link handling.